var/spack/repos/builtin/packages/freetype/package.py

```diff
@@ -47,6 +48,13 @@ def headers(self):
         headers.directories = [self.prefix.include.freetype2]
         return headers
 
+
+class CMakeBuilder(CMakeBuilder):
+    def cmake_args(self):
+        return [self.define("FT_WITH_ZLIB", True), self.define("FT_WITH_BZIP2", True)]
```

Comment:
1. Why is `FT_WITH_ZLIB` true? It's explicitly disabled for Autotools. We also do not have a `zlib` dependency.
2. What about the other options? This should probably enable/disable the same features as the Autotools variant.

Reply:
This was a mistake I made during a (very old) previous iteration. It was _supposed_ to be `FT_DISABLE_ZLIB` set to ON. This is fixed on the current state.